### ISSUE TITLE
Enhanced image embedding format

### ISSUE THREAD

**Issue by zkn365** [pymupdf/RAG#128]
The following format cannot handle whitespace in the filename:
```
![](file name with whitespace.png)   nothing shown
```
Please consider the following format, which can display the image if there is whitespace in the filename:
```
![](<file name with whitespace.png>)
```

**Comment by JorjMcKie** [comment]
Thanks for the hint. Will include this change in the next version.

**Comment by JorjMcKie** [comment]
Fixed in version 0.0.15.
